### PR TITLE
Standardized geekdoc casing, documented weight

### PR DESCRIPTION
**Pull request by jamesbraza**
Documented weight sizing and negative being allowed

### CODE REVIEW

File: exampleSite/content/en/usage/configuration.md [hunk 1]
```diff
@@ -269,7 +269,8 @@ params:
 # Set type to 'posts' if you want to render page as blogpost
 type = "posts"
 
-# Set page weight to re-arrange items in file-tree menu.
+# Set page weight to re-arrange items in file-tree menu. Higher weight means
```
Comment (jamesbraza) [inline exampleSite/content/en/usage/configuration.md:272]: Are `float` weights allowed?

Comment (xoxys) [reply exampleSite/content/en/usage/configuration.md:272]: This is a Hugo option, and I don't want to duplicate the documentation for it. If necessary, we can add a link to https://gohugo.io/content-management/front-matter/?search-input=weight#front-matter-variables.

File: exampleSite/content/en/_includes/_index.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 ---
-GeekdocHidden: true
+geekdocHidden: true
```
Comment (xoxys) [inline exampleSite/content/en/_includes/_index.md:2]: The casing is intended. Parameters in Hugo are case-insensitive by default, so casing technically doesn't matter, but we decided to use lowercase in the config files and uppercase for occurrences in templates and front matters.

**Review by xoxys — CHANGES_REQUESTED** [review]

File: exampleSite/content/en/usage/configuration.md (hunk 1 above)
Comment (jamesbraza) [reply exampleSite/content/en/usage/configuration.md:272]: Okay, I will update to include this link tomorrow

File: exampleSite/content/en/_includes/_index.md (hunk 2 above)
Comment (xoxys) [reply exampleSite/content/en/_includes/_index.md:2]: Hmm, now I'm not sure either :smile:  Let me think about it a bit

File: exampleSite/content/en/usage/configuration.md (hunk 1 above)
Comment (jamesbraza) [reply exampleSite/content/en/usage/configuration.md]: Done, lemme know if you have further requests here

File: exampleSite/content/en/_includes/_index.md (hunk 2 above)
Comment (jamesbraza) [reply exampleSite/content/en/_includes/_index.md:2]: Fwiw, I think having one naming convention (dromedary case e.g. `geekdocHidden`) is the right call.  The less theme-specific naming conventions to learn about, the better

Comment (xoxys) [reply exampleSite/content/en/_includes/_index.md:2]: As there were a lot more occurrences, I have created a dedicated PR for this. https://github.com/thegeeklab/hugo-geekdoc/pull/647

After it got merged, please rebase your PR.

File: exampleSite/content/en/usage/configuration.md [hunk 3]
```diff
@@ -265,11 +265,13 @@ params:
...
 
-```Toml
+```toml
 # Set type to 'posts' if you want to render page as blogpost
 type = "posts"
 
-# Set page weight to re-arrange items in file-tree menu.
+# Hugo predefined front matter variable, to re-arrange items in file-tree menu
+# SEE weights section of
+# https://gohugo.io/content-management/front-matter/?search-input=weight#predefined
```
Comment (xoxys) [inline exampleSite/content/en/usage/configuration.md:274]:
```suggestion
# https://gohugo.io/content-management/front-matter/#predefined
```

**Review by xoxys — APPROVED** [review]

File: exampleSite/content/en/_includes/_index.md (hunk 2 above)
Comment (jamesbraza) [reply exampleSite/content/en/_includes/_index.md]: Rebase complete 👌

**Review by xoxys — APPROVED** [review]

**Comment by xoxys** [comment]
Thanks!

**Pull request merged**